misc/custom-flows-environment-variables/env0.yml

```diff
@@ -5,7 +5,13 @@ deploy:
     terraformPlan:
       before:
         - echo USER_NAME=user >> $ENV0_ENV
+        - echo FULL_NAME_WITH_WHITESPACE=\"John Doe\" >> $ENV0_ENV
         - ./export-user-id.sh
       after:
         - echo $USER_NAME
+        - echo $FULL_NAME_WITH_WHITESPACE
         - ./print-user-id.sh
+        - echo "Asserting expected values..."
+        - "[ $USER_NAME == user ]"
+        - "[ \"$FULL_NAME_WITH_WHITESPACE\" == \"John Doe\" ]"
+        - "[ $USER_ID == user_id ]"
```

Comment:
🟡 
why does the condition need quotation marks?

Reply:
![image](https://user-images.githubusercontent.com/83922349/169770556-f871a93c-5271-4019-86fe-c9f8942ce5ca.png)